jenkinsapi_tests/unittests/test_build.py

```diff
@@ -101,6 +101,14 @@ def test_get_params(build):
     assert params == expected
 
 
+def test_get_build_url(build):
+    expected = "http://foo/1"
+    build._data = {"url": "http://foo/1"}
+    url = build.get_build_url()
+    if not url == expected:
```

Comment:
Please change this to pytest style assert.

Reply:
The code coverage goes red if i do that 
with error like this 
Avoid asserts
Problematic code:
assert condition
Correct code:
if not condition:
    raise AssertionError()

Reply:
I switched that check off, so please reformat code to be the same looking like the rest of test suite.

Thanks!

Reply:
done. Can you please look this one and 634 , i plan to use and this changes soon ..